components/Buttons/src/ColorThemer/MDCOutlinedButtonColorThemer.h

```diff
@@ -25,7 +25,8 @@
 @interface MDCOutlinedButtonColorThemer : NSObject
 @end
 
-@interface MDCOutlinedButtonColorThemer (ToBeDeprecated)
+__deprecated_msg("Please use [MDCButton applyOutlinedThemeWithScheme:] instead. (Note: Color theming is no longer available as an independent API.")
```

Comment:
Deprecating the category doesn't cause warnings to be generated by usages of `MDCOutlinedButtonColorThemer` (see MDCAlertColorThemer.m, line 42). This PR should remove the `@interface MDCOutlinedButtonColorThemer (ToBeDeprecated)` category declaration and apply the deprecation warning directly to the `@interface` declaration above on line 25.

Reply:
Done.